### ISSUE TITLE
upgrading minor challenge issues across the board with a batch script

### ISSUE THREAD

**Issue by nikrb** [freeCodeCamp/freeCodeCamp#34836]
formatting challenges has been updated (#34747) so `<br>` tags should no longer be required. Can anyone think of edge cases where a `<br>` may still be required?

variables declared with `var` could also be changed to const - or let if there is (re)assignment in the body.
@ezioda004 pointed out there are challenges in ES6 section specifically aimed at changing var to const/let so miss that section.

**Comment by ezioda004** [comment]
Just to clarify -  I meant, the challenges before the ES6 challenge [Explore Differences Between the var and let Keywords](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/es6/explore-differences-between-the-var-and-let-keywords) *should not* be modified with `let/const`.

**Comment by nikrb** [comment]
I have a [script](https://github.com/nikrb/change-var-to-const-let.git) that converts var defs to const, or let if reassigned.
[160 english md files](https://github.com/nikrb/change-var-to-const-let/blob/master/outlist.txt) are changed, though a number of directories are skipped for various reasons.
A dozen or so files have issues with client app and curriculum tests because of seed code and/or solutions.
The conversion itself causes 5 failures due to bad syntax constructs.

Suggest we run this when the number of pr's returns to a handlable level.
Script doesn't currently remove `<br>` tags but this is easy to add.

**Comment by RandellDawson** [comment]
> Suggest we run this when the number of pr's returns to a handlable level.

I second this recommendation.  There are a couple of PRs we are going to wait on creating (mass find/replace) PRs which affect 100s of files, so we have decided to wait until we get below 500 open PRs to avoid other PRs stacking up behind them as we clear through eariler PRs to prevent mass chaining of merge conflicts.

I think it is a great idea and some of this logic could be added to the Probot to catch these kinds of things when new PRs are created.  When it gets closer to time to create the actual PR, we should revisit this issue to make sure we have everything covered.

**Comment by moT01** [comment]
Is this something we want to try and tackle now? Looks like the number of PR's has been pretty low for a while.

**Comment by RandellDawson** [comment]
Probably is fine now.  We just do not want to use `let` and `const` until after the ES6 section is introduced.  Anything before it should remain `var`.

**Comment by moT01** [comment]
Closing as stale. If you come across this @nikrb, you should create a new issue and we can discuss running your script again. Thanks and happy coding 🎉
